libs/hwes/eventstoredb/entity_event.go

```diff
@@ -0,0 +1,66 @@
+package eventstoredb
+
+import (
+	"context"
+	"fmt"
+	"telemetry"
+
+	"github.com/EventStore/EventStore-Client-Go/v4/esdb"
+	"github.com/google/uuid"
+	"google.golang.org/protobuf/proto"
+	"hwes"
+)
+
+// Extending entityAggregate to override GetTypeID down below
+type entityAggregate struct {
```

Comment:
why do we need this type? why do we not use normal aggregates for everything? then we can switch to an event-sourced approach *really* easily if we want to

Reply:
why? are the events not the same as when we'd use then to construct projections with them? why would we want to disregard all existing events?

wasn't the point to have the write-part of the ES approach ready to use? we just don't *source* from these events?

Reply:
_We discussed this via audio_

- [  ] Use aggregate type to differentiate between event sourced and non event sourced event streams